source/_integrations/weather.brightsky.markdown

```diff
@@ -0,0 +1,57 @@
+---
+title: "Bright Sky"
+description: "Instructions on how to integrate Bright Sky within Home Assistant."
+logo: dwd.png
```

Comment:
Please add the logo to our [Brands repository](https://github.com/home-assistant/brands)